source/cfn/templates/main.template

```diff
@@ -282,12 +368,10 @@ Resources:
       TimeoutInMinutes: 60
       Parameters:
         AppName: !FindInMap [Solution, Constants, AppName]
-        AvailabilityZone0: !GetAtt VpcStack.Outputs.AvailabilityZone0
-        AvailabilityZone1: !GetAtt VpcStack.Outputs.AvailabilityZone1
```

Comment:
You did a fix in the neptune stack, but you need to fix this one too? Something like `!If [CreateVpc, ...`

Reply:
Ah yes, good point!

Reply:
*done